src/tests/utils.ml

```diff
@@ -142,7 +142,7 @@ let a2 contents_root =
 
 let not_very_random n =
   let i = Random.int n in
-  if i > Pervasives.(n / 2) then 0 else i
+  if i > Stdlib.(n / 2) then 0 else i
```

Comment:
Do we still run opam-rt on pre-4.07 on any of the old branches?? (I kinda hope not!)

Reply:
No, the only two checks that run it are using 4.14: https://github.com/ocaml/opam/blob/f93526d411e6f7ef0b4c9d5190ff6d98cf624d28/.github/workflows/main.yml#L358